docs/markdown/ios/card/card.pins.md

```diff
@@ -55,10 +56,13 @@ People use Apple’s accessibility features, such as reduced transparency, Voice
 *1. Image one / 2. Image two / 3. Image three*
 
 ## Variants
-The Card.Boards preview block utilizes the [Gestalt Board and Section Figma component](http://pinch.pinadmin.com/section-and-board-component-figma-iOS). The Card.Boards preview block accepts all of Board and Sections variants except for CTAs. Optional elements include: lock icon, plus icon, pretext, Card.Boards name, collaborators and subtext.
+
+Card.Pins is a preview block utilizing two of the Pin and Section components to present a collection of imagery. The Card.Pins preview block accepts all of Pin and Sections variants except for CTAs. Optional elements include: lock icon, plus icon, pretext, Card.Pins name, collaborators and subtext.
```

Comment:
The previous copy here was referencing Card.Boards, so seemed to be incorrect. Not sure if this new copy is factually correct

Reply:
sorry, another cut and paste error. It's probably good to have a Variants intro paragraph, but the original coda doc doesn't have one. The embeds are also incorrect. https://coda.io/d/_dqIC9SDPsU9/Pins-Preview-Block_suJtg#_lukRk